languages/js/src/Oso.ts

```diff
@@ -258,13 +258,18 @@ export class Oso<
   async authorizedQuery(
     actor: Actor,
     action: Action,
-    resourceCls: Class<Resource>
+    resourceCls: Class<Resource> | string
```

Comment:
This is a nice quality of life improvement. In many cases it's easier to just provide a string for the class name (like with Prisma)